### ISSUE TITLE
LTE Modem configuration

### ISSUE THREAD

**Issue by tsh2001** [Ysurac/openmptcprouter#3102]
## Expected Behavior
I have 2 4G LTE Dongle and a ethernet modem to connect to the router. i connected one of the Dongle to the RPI4 USB port and another to a USB Hub.

## Current Behavior
I cant seem to get it to work. its stating Duplicate Network interface for the ethernet modem and it cannot detect the 2nd LTE Dongle

<img width="792" alt="image" src="https://github.com/Ysurac/openmptcprouter/assets/140482485/f98b4e0b-bc44-4e8b-86a8-ac104a84a500">

## Specifications

  - OpenMPTCProuter version: 0.59.1
  - OpenMPTCProuter VPS version: 0.1028
  - OpenMPTCProuter VPS provider: DigitalOcean
  - OpenMPTCProuter platform: RPI4

**Comment by Ysurac** [comment]
It's working with only "modem2" connected ?

**Comment by tsh2001** [comment]
Hi, 
With only 'modem2' connected, it still state Network Interface Duplicated.

Additionally, I tried using another Dongle, connected it to the RPI along with one of the 4G LTE Dongle. The router recognise the new dongle and set it as wwan0. However, it doesn't seem to be connected to the internet. I have set it as DHCP 'wwan0' but its stating no IP address or Gateway found.

![photo_2024-01-04_14-08-40](https://github.com/Ysurac/openmptcprouter/assets/140482485/5ffd1874-20b2-4e34-8a9c-ee2258cf8c07)
